lib/galaxy/tools/deps/resolvers/conda.py

```diff
@@ -0,0 +1,128 @@
+"""
+This is still an experimental module and there will almost certainly be backward
+incompatible changes coming.
+"""
+
+
+import os
+
+from ..resolvers import DependencyResolver, INDETERMINATE_DEPENDENCY
+from ..conda_util import (
+    CondaContext,
+    CondaTarget,
+    install_conda,
+    is_conda_target_installed,
+    install_conda_target,
+    build_isolated_environment,
+)
+
+DEFAULT_ENSURE_CHANNELS = "r,bioconda"
```

Comment:
:+1:, that way the whole point of "you can prefer your own lab's X over devteam over IUC" will actually be really possible